packages/react-day-picker/src/types/DayPicker.ts

```diff
@@ -184,7 +180,7 @@ export interface DayPickerProps {
   /**
    * Add modifiers to the matching days.
    */
-  modifiers?: CustomModifiers;
+  modifiers?: DayModifiers;
```

Comment:
I've introduced a new type, `DayModifiers`, that accepts matcher as single value - as opposite of `CustomModifiers` that accept only _array_ of Matchers. 

`DayModifiers` only surfaces in the DayPicker props – in our code it is transformed to a `CustomModifiers` object (hence, only arrays).